### DISCUSSION TITLE
Being redirected to torus helpdesk sometimes

### DISCUSSION THREAD

**Discussion by Haris565** [Web3Auth/Web3Auth#745] [Help]
I have integrated Google and Apple Signin in my react native app some time signin flow works smooth and sometime it redirects the user to torus helpdesk i tried everything the issue still persist can anyone guide me how to disable this helpdesk or any way out

**Comment by BboyStatix** [comment]
@Haris565 are you still facing this issue? Is it possible for you to provide some error logs?

**Comment by Haris565** [comment]
Hi
Yes im still facing this issue and there is no error logs i have
implemented web3auth react native sdk

**Reply by shahbaz17** [reply]
Hey @Haris565 

If you're using Chrome, then going to `chrome://inspect` will show the devices attached. Read more about debugging [here](https://reactnative.dev/docs/debugging).

Every time there's a runtime error, helpdesk pop-up will be shown.
